addons/stock/models/stock_move_line.py

```diff
@@ -28,9 +28,8 @@ class StockMoveLine(models.Model):
     product_id = fields.Many2one('product.product', 'Product', ondelete="cascade", check_company=True, domain="[('type', '!=', 'service'), '|', ('company_id', '=', False), ('company_id', '=', company_id)]")
     product_uom_id = fields.Many2one('uom.uom', 'Unit of Measure', required=True, domain="[('category_id', '=', product_uom_category_id)]")
     product_uom_category_id = fields.Many2one(related='product_id.uom_id.category_id')
-    product_qty = fields.Float(
-        'Real Reserved Quantity', digits=0,
-        compute='_compute_product_qty', inverse='_set_product_qty', store=True)
+    product_qty = fields.Float('Real Reserved Quantity', compute='_compute_product_qty',
```

Comment:
you should keep the new line after the `(`, not sure if it is PEP compliant.
```
product_qty = fields.Float(
     'Real Reserved Quantity', compute='_compute_product_qty',
     inverse='_set_product_qty', store=True)
```

Reply:
Yep, noted!

Reply:
@amoyaux Did you notice the removal of digits=0?  Seems even a schema change to me?

Reply:
@jco-odoo digits=0 then no rounding? I don't see its role in the field.

Reply:
odoo/odoo#67129